bsp/chips/cc1200/cc1200.h

```diff
@@ -415,52 +410,61 @@ void cc1200_spiReadRam    (uint16_t addr,  cc1200_status_t* statusRead, uint8_t*
 static const registerSetting_t preferredSettings[] = {
     {CC1200_IOCFG2,         0x06}, //it was 0x13 before. 
     {CC1200_IOCFG0,         0x06},
-    {CC1200_SYNC_CFG1,      0xA9},
+    {CC1200_SYNC_CFG1,      0xE5},  // 0xA9 before. Now, dual sync search. 
+    {CC1200_SYNC_CFG0,      0x23},
+    {CC1200_SYNC3,          0x6E},  //jmms
+    {CC1200_SYNC2,          0x4E},  //jmms
+    {CC1200_SYNC1,          0x90},  //jmms
+    {CC1200_SYNC0,          0x4E},  //jmms
+    {CC1200_DEVIATION_M,    0x47},
     {CC1200_MODCFG_DEV_E,   0x0B},
+    {CC1200_DCFILT_CFG,     0x56},
     {CC1200_PREAMBLE_CFG1,  0x30},
     {CC1200_PREAMBLE_CFG0,  0x8A},
     {CC1200_IQIC,           0xC8},
-    {CC1200_CHAN_BW,        0x10},
+    {CC1200_CHAN_BW,        0x84},
     {CC1200_MDMCFG1,        0x42},
     {CC1200_MDMCFG0,        0x05},
-    {CC1200_SYMBOL_RATE2,   0x8F},
-    {CC1200_SYMBOL_RATE1,   0x75},
-    {CC1200_SYMBOL_RATE0,   0x10},
+    {CC1200_SYMBOL_RATE2,   0x94},
+    {CC1200_SYMBOL_RATE1,   0x7A},
+    {CC1200_SYMBOL_RATE0,   0xE1},
     {CC1200_AGC_REF,        0x27},
-    {CC1200_AGC_CS_THR,     0xE4},
+    {CC1200_AGC_CS_THR,     0xF7},
     {CC1200_AGC_CFG1,       0x00},
     {CC1200_AGC_CFG0,       0x90},
+    {CC1200_FIFO_CFG,       0x00},
     {CC1200_SETTLING_CFG,   0x03},
     {CC1200_FS_CFG,         0x12},
     {CC1200_WOR_CFG0,       0x20},
-    {CC1200_WOR_EVENT0_LSB, 0xC3},
-    {CC1200_PKT_CFG2,       0x00},
- //   {CC1200_PKT_CFG2,       0x20},
- //   {CC1200_PKT_CFG1,       0x01},  //added by jmms
+    {CC1200_WOR_EVENT0_LSB, 0x96},
+ //   {CC1200_PKT_CFG2,       0x00},
+    {CC1200_PKT_CFG2,       0x24},  //jmms
```

Comment:
Please don't sign every line with jmms. IF we all did thins, it would be a mess. You are the author of thta file, so everyone assumes you have either written, or vetted every line in that file.